examples/pio_blink/rpx0xx_pio_blink/blink.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2023 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#include <errno.h>
+#include "pio/pio.h"
+#include "blink.hex.pio.h"
+
+static void pio_blink_init_pins(pio_t pio, pio_sm_t sm, gpio_t pin)
+{
+    pio_gpio_init_t init = {
+        .gpio_direction = 0x01,
+        .gpio_base = pin,
+        .gpio_count = 1
+    };
+    pio_sm_set_set_pins_init(pio, sm, &init);
+}
+
+pio_program_t pio_blink_export_program(void)
+{
+    return pio_blink_create_program();
+}
```

Comment:
hmm is that rename needed? or can it be 4 lines less here?

Reply:
Because `pio_blink_create_program()` is a `static inline` function in a header file and thus only exists in the file where it is included, yes.
Either that way or an `INCLUDES += $(CPU)_pio_blink` in the application Makefile and include the header in `main.c`.
I find the latter option less clean because the header content depends on what PIO CPU implementation is used.
It could look different for other CPUs implementing PIO, maybe including other CPU headers or even vendor stuff.
So I think it is good if `main.c` must not know about it.

Reply:
I suppose `pio_blink_create_program()` is from the generated code then?
Since this is an example, better add a comment about that.